topology/Thomas Jefferson National Accelerator Facility/JLAB/JLab-Farm.yaml

```diff
@@ -43,6 +44,7 @@ Resources:
           Name: Kurt Strosahl
     Description: Submit node for JLab
     FQDN: scosg20.jlab.org
+    DN: /CN=scosg16.jlab.org
     ID: 1134
```

Comment:
I realize this ID is not part of the change, but 1134 is already used by a Lamar-Cluster resource.

Reply:
We test for name uniqueness and ID presence, but not ID uniqueness. I wonder why...

Reply:
Oh huh.  Yeah i suppose we should add a unique ID check for the different types of things, ey?

Reply:
I don't think we can assume that: the registered FQDN may correspond to one of the cert's SANs.

Reply:
Ah ok.

Also:

>It's concerning that our CI doesn't catch this :(

>We test for name uniqueness and ID presence, but not ID uniqueness. I wonder why...

Apparently there is a ticket for this - SOFTWARE-3287 specifies a few new unique ID requirements